### PR TITLE
Short explanation of the open(SUSE) development/maintenance process with links

### PR DESCRIPTION
**Pull request by ancorgs**
Although the tools in this repository are created to assist in the development and maintenance of the (open)SUSE distributions, there are no links to any document explaining how those processes work.

@teclator compiled several links in [this short text](https://gist.github.com/teclator/0cba3711afd8249fe605490cd238a03d). This pull request is an alternative version of that same text with (more or less) the same links.

This PR is somehow complementary to #2604,

### CODE REVIEW

**Comment by ancorgs** [comment]
If we merge this, we should likely then ask @lkocman to add a paragraph and some links for Leap and @sysrich to do the same for MicroOS.

File: docs/processes.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+Maintenance requests (similar to the submit requests created during development) are created to
+release an update of a package for an already released distribution. Those requests also go through
+a review process assisted by some of the tools in this repository. Similar to the staging workflow,
+in which one or more packages are verified in a staging project before their submit requests can be
+accepted into the GA project, the packages from one or several maintenance request are verified in a
+so-called incident project before accepting them in the corresponding update project.
+
+Despite the similarities between the concept of maintenance incidents and the staging workflow, the
+former is implemented by separate mechanisms not based on any of the tools in this repository.
```
Comment (teclator) [inline docs/processes.md:102]: Maybe we could link the [OBS maintenance support](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.maintenance_setup.html) page here.

File: docs/processes.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+like the [Origin Manager](./origin-manager.md) to verify aspects that are not relevant for
+Tumbleweed.
+
+The following [SUSE-internal
+document](https://confluence.suse.com/display/projectmanagement/Product+Handbook) offers all kind of
+details about the processes involved in the development of SLE and all its associated products.
+
+## openSUSE Leap Development
+
+Starting with version 15.3, openSUSE Leap shares its base binary packages with the corresponding
```
Comment (teclator) [inline docs/processes.md:71]:
```suggestion
Starting with version [15.3](https://en.opensuse.org/Portal:15.3), openSUSE Leap shares its base binary packages with the corresponding
```

File: docs/processes.md (hunk 1 above)
Comment (ancorgs) [reply docs/processes.md]: We definitely should! I have overlooked that one. Good catch.

**Comment by okurz** [comment]
+1

you are doing really cool stuff here lately :)

**Pull request merged**